### ISSUE TITLE
small parser bug/ feature request when using multiple loops with { } ?

### ISSUE THREAD

**Issue by Reiner030** [saltstack/salt-vim#36]
Hello,

I got a file with a nice onliner for setting variables from pillar to enhance like:
```
{%- do environments[environment].update ({ 'jenkins': { 'port': '1' + port['environment'][environment]['port'] + '00' + '6' }}) %}
```
and the closing loops written as ```}}``` are marked red because the parser identifies it as closing saltstack tag which is missing the opening tag instead of 2 independent ending loop marker.

If this is not too complicated it would be nice there could a fix ?

Bests

**Comment by basepi** [comment]
Hrm, I'm not sure what the fix would look like here, because in general those are closing jinja variable tags. Have you tried putting a space between the two curly braces?

**Comment by Reiner030** [comment]
yes... and it works. I suggest the code owner then to put there better spaces between them ^^... 
Was only the idea that the parser perhaps could easy handle such situation, too. 
If its not easy then feel free to close this thread ;)

**Comment by basepi** [comment]
I'm going to leave it open for the moment. I don't have any bright ideas but maybe someone else will.
